Source/processcontainers/CMakeLists.txt

```diff
@@ -96,6 +100,49 @@ elseif (PROCESSCONTAINERS_CRUN)
                 PRIVATE
                 libcrun::libcrun
         )
+elseif (PROCESSCONTAINERS_DOBBY)
+        find_package(jsoncpp REQUIRED)
```

Comment:
Thunder already has a JSON Serializer/Deserializer, Why do you include yet another JSON library. This adds an unneeded dependency to the Thunder libraries. Please just use Core (which is already part of this setup)